Datahub.Core/Services/EmailNotificationService.cs

```diff
@@ -538,7 +538,7 @@ public async Task SendOnboardingConfirmations(OnboardingParameters parameters, b
         public async Task SendOnboardingMetadataEditRequest(OnboardingParameters parameters)
         {
             var parametersDict = BuildOnboardingParameters(parameters);
-            var subject = $"Please complete the details for your DataHub Initiative – {parameters.App.Product_Name}";
+            var subject = $"Please complete the details for your DataHub Initiative";
```

Comment:
Just realized we have not figured out the French version :-\

Reply:
That's right, we should add the French under the English body and rework the subject.